domain/src/main/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/transfers/Transfer.java

```diff
@@ -0,0 +1,180 @@
+            return this;
+        }
+
+        /**
+         * Sets field toStopId value and returns this
+         *
+         * @param toStopId a stop or station where a connection between routes ends
+         * @return builder for future object creation
+         */
+        public TransferBuilder toStopId(@NotNull final String toStopId) {
+            this.toStopId = toStopId;
+            return this;
+        }
+
+        /**
+         * Sets field transferType value and returns this
+         *
+         * @param transferType indicates the type of connection for the specified
+         * @return builder for future object creation
+         */
+        public TransferBuilder transferType(@NotNull final Integer transferType) {
+            this.transferType = TransferType.fromInt(transferType);
+            this.originalTransferTypeInteger = transferType;
+            return this;
+        }
+
+        /**
+         * Sets field minTransferTime value and returns this
+         *
+         * @param minTransferTime amount of time, in seconds, that must be available to permit a transfer between
+         *                        routes at the specified stops
+         * @return builder for future object creation
+         */
+        public TransferBuilder minTransferTime(@Nullable final Integer minTransferTime) {
+            this.minTransferTime = minTransferTime;
+            return this;
+        }
+
+        /**
+         * Returns an entity representing a row from transfers.txt if the requirements from the official GTFS
+         * specification are met. Otherwise, method returns an entity representing a list of notices.
+         *
+         * @return entity representing a row from transfers.txt if the requirements from the official GTFS specification
+         * are met. Otherwise, method returns an entity representing a list of notices.
+         */
+        public EntityBuildResult<?> build() {
+            noticeCollection.clear();
+
+            if (fromStopId == null || toStopId == null || transferType == null ||
+                    (minTransferTime != null && minTransferTime < 0)) {
+
+                if (fromStopId == null) {
+                    noticeCollection.add(new MissingRequiredValueNotice("transfers.txt",
+                            "from_stop_id", fromStopId + ";" + toStopId));
+                }
+                if (toStopId == null) {
+                    noticeCollection.add(new MissingRequiredValueNotice("transfers.txt",
+                            "to_stop_id", fromStopId + ";" + toStopId));
+                }
+                if (!TransferType.isEnumValueValid(originalTransferTypeInteger)) {
+                    noticeCollection.add(new UnexpectedEnumValueNotice("transfers.txt",
+                            "transfer_type", fromStopId + ";" + toStopId,
+                            originalTransferTypeInteger));
+                }
+                if (minTransferTime != null && minTransferTime < 0) {
```

Comment:
I don't think `0` is a valid value for `minTransferTime` either. The [spec says](https://gtfs.org/reference/static#transferstxt):

>Amount of time, in seconds, that must be available to permit a transfer between routes at the specified stops. The min_transfer_time should be sufficient to permit a typical rider to move between the two stops, including buffer time to allow for schedule variance on each route.

So it seems like it should be 1 at a minimum. This field is also a candidate for another threshold that could be configurable for what the lower possible value is that makes sense. For example, I would like that most min_transfer_times are greater than 30 seconds, but I don't know that for sure.

Reply:
Good point, modifying this keeping in mind that the field could be configurable (via a .json file for example). 
Opening an issue on this matter for the future.

Reply:
See https://github.com/MobilityData/gtfs-validator/issues/181

Reply:
This field has been made user configurable in 373cc25fbeb4ca9a8491885f7707a02e51591efe